espymetrics/dal/sqlite_dal.py

```diff
@@ -13,25 +13,116 @@
 
 * Not human readable - Binary format makes it harder to read/grep
 * More complex - More logic = more work = more time/resources?
-
 """
 
+import os
+import sqlite3
+
+import boltons
 
-_MISSING = object()
 SEP = '$'  # '$' is valid in sqlite column names, no escaping required
 
+TABLE_NAME = 'on_import_data'
+CREATE_TABLE = ('CREATE TABLE IF NOT EXISTS ' + TABLE_NAME +
+                '(id integer primary key asc, ' +
+                'hostfqdn, hostname, python$argv, python$bin, ' +
+                'python$build_date, python$compiler, python$have_readline, ' +
+                'python$have_ucs4, python$is_64bit, python$version, python$version_full, ' +
+                'time$utc_epoch real, time$std_utc_offset real, ' +
+                'uname$0, uname$1, uname$2, uname$3, uname$4, uname$5, ' +
+                'username, uuid)')
+
+total_count = 0
+
+sqlite3.enable_callback_tracebacks(True)
+
 
 class SQLiteDAL(object):
     _extension = '.db'
 
+    __file_path = None
```

Comment:
Unlike Java and C++, class members do not need class-level declarations. This default value is unnecessary.